src/tests/multihost/alltests/readme.rst

```diff
@@ -32,6 +32,7 @@ Following are the pytest markers used
 * krbldapconnection: Tests related to kerberos ldap connection
 * passwordcheck: Tests related to password check while updating password of user
 * proxy: Tests related to sssd-proxy
+* proxy_provider : Tests related to Proxy Provider
```

Comment:
Line 34 is used as a marker, so we dont need 'proxy_provider' marker

Reply:
Done